# manipulator:
#   regions:
#     place_region:
#       min_limits:
#         [0.90, -0.07, 0.46, 0.0, 0.0, 0.0]
#       max_limits:
#         [0.90, 0.45, 0.46, 0.0, 0.0, 0.0]

# l_arm:
#   regions:
#     place_region:
#       min_limits:
#         [0.40, 0.16, 1.2, 3.10, -0.96, -3.14159]
#       max_limits:
#         [0.50, 0.26, 1.6, 3.14159, -0.92, -3.10]

# manipulator_1:
#   regions:
#     pick_region:
#       min_limits:
#         [-0.4, 0.55, 1.15, 0, 0, 0.0]
#       max_limits:
#         [0.4, 1.0, 1.30, 0, 0, 0.0]
#     place_region:
#       min_limits:
#         [0.4, -0.25, 1.1, 0.0, 0.0, -0.1]
#       max_limits:
#         [0.9, 0.25, 1.1, 0.0, 0.0, 0.1]

manipulator_1:
  regions:
    pick_region:
      min_limits:
        [-0.12, -1.06, 1.08, 0.0, 0.0, 0.0]
      max_limits:
        [0.12, -0.82, 1.12, 0.0, 0.0, 0.0]
      discretization:
        [0.02, 0.02, 0.02, 0.0174533, 0.0174533, 0.0174533]
      goal:
        [0.1, -0.84, 1.10, 0.0, 0.0, 0.0]

    place_region:
      min_limits:
        [0.80, -0.06, 1.12, 0.0, 0.0, 0.0]
      max_limits:
        [1.00, 0.06, 1.14, 0.0, 0.0, 0.0]
      discretization:
        [0.005, 0.005, 0.005, 0.00872665, 0.00872665, 0.00872665]
      goal:
        [0.818, -0.052 , 1.13, 0.0, 0.0, 0.0]